calling-apis/managing-asynchronous-calls.md

```diff
@@ -309,11 +334,17 @@ Which of the following most accurately describes where we print the results from
 ### !end-challenge
 <!-- prettier-ignore-end -->
 
-This solution successfully uses the results of one API call to make another API call, but it has a few drawbacks:
+Raffy chose to create a new function `getLocationFromQuery` to manage calling the existing functions `findLatitudeAndLongitude` and `findLocation`. He could have called the second method directly from inside the first with `.then` chaining, but doing so would have linked these two functions together, making it harder to use one without the other.
+
+To use `findLatitudeAndLongitude` from another function and pass on its `response` data to another network call, we needed to make a couple updates in `findLatitudeAndLongitude`: 
+- `return` the result of the `axios` call which returns the promise chain the axios call creates.
```

Comment:
I think this can be clearer, but I'm a little concerned about putting an exact line number in the lesson that could get out of sync. How do you feel about:

```suggestion
- `return` the result of the `axios` call. This returns the promise chain created by the network call and chained `.then` blocks. Look for the `return` statement in `findLatitudeAndLongitude` that starts with `return axios.get(...`.
```